libcalico-go/lib/ipam/ipam_block.go

```diff
@@ -378,7 +426,7 @@ func sanitizeHandle(handleID string) string {
 	return strings.Split(handleID, "\r")[0]
 }
 
-func (b *allocationBlock) releaseByHandle(handleID string) int {
+func (b *allocationBlock) releaseByHandle(handleID string, opts ReleaseOptions) int {
```

Comment:
nit: now that we have all the release information in the `ReleaseOptions`, do we need to pass in the explicit `handleID` anymore?